### ISSUE TITLE
Open second fman window on macos

### ISSUE THREAD

**Issue by jonaskello** [fman-community/fman#500]
I want to open a second fman window in macos. For Chrome I can just right click the icon in the dock and select "New window". Right clicking on fman does not show that menu item. I also searched the command but found nothing that will open a new window. I can however typ `fman .` in the terminal and that will open a second instance of fman with a new window. I guess this should be easy and I feel a bit stupid for not finding a way to do it :-). But I see that fman have no "Window" menu like other apps on mac so perhaps there is no multi-window support yet?

**Comment by mherrmann** [comment]
Yeah, there's no multi-window support yet on macOS I'm afraid. You could [write a plugin](https://fman.io/docs/plugins-introduction) that executes `/Applications/fman.app/Contents/MacOS/fman`.

The problem of the above workaround is that fman appears in macOS's task bar twice. Also, the common macOS shortcut <kbd>Cmd</kbd>+<kbd>`</kbd> does not work to switch between windows. The goal of this issue should thus be to implement this properly.

**Comment by albert-ying** [comment]
> Yeah, there's no multi-window support yet on macOS I'm afraid. You could [write a plugin](https://fman.io/docs/plugins-introduction) that executes `/Applications/fman.app/Contents/MacOS/fman`.
> 
> The problem of the above workaround is that fman appears in macOS's task bar twice. Also, the common macOS shortcut Cmd+` does not work to switch between windows. The goal of this issue should thus be to implement this properly.

Is there any better workaround of this?  Although it can be switched cmd+tab, it is kind of frustrating since the icon looks the same. It would be great if cmd+` can be used. Or is there way to change the icon of second window to make them distinguishable?
